joinfaces-starters/icefaces-spring-boot-starter/pom.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.joinfaces</groupId>
+        <artifactId>joinfaces-starters</artifactId>
+        <version>3.2.5-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>icefaces-spring-boot-starter</artifactId>
+    <name>ICEfaces Spring Boot Starter</name>
+    <description>ICEfaces Component JSF Spring Boot Starter</description>
+
+    <properties>
+        <main.basedir>${basedir}/../..</main.basedir>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.joinfaces</groupId>
+            <artifactId>jsf-spring-boot-starter</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.icefaces</groupId>
+            <artifactId>icefaces</artifactId>
```

Comment:
Should `icefaces-spring-boot-starter` pull `icefaces-ace` dependency too? Our `primefaces-spring-boot-starter` pull `primefaces-extensions`.